configs/abcxyz/infra.yaml

```diff
@@ -1,12 +1,12 @@
 # abcxyz/infra.yaml
 - if: 'assertion.workflow_ref.startsWith("abcxyz/infra/.github/workflows/terraform.yml")'
-  respositories:
-    - 'abcxyz/github-metrics-aggregator'
-    - 'abcxyz/github-token-minter'
-    - 'abcxyz/guardian'
-    - 'abcxyz/infra'
-    - 'abcxyz/jvs'
-    - 'abcxyz/lumberjack'
-    - 'abcxyz/pmap'
+  repositories:
+    - 'github-metrics-aggregator'
+    - 'github-token-minter'
+    - 'guardian'
+    - 'infra'
+    - 'jvs'
+    - 'lumberjack'
+    - 'pmap'
```

Comment:
This is an interesting change. How does org come into play now?

Reply:
The org is part of the installation id so you can't actually pick a repository outside of `abcxyz`. Since we only support a single install we don't need to specify.

Reply:
Got it. I guess that makes sense.